paltas/Configs/config_handler.py

```diff
@@ -24,8 +25,22 @@
 # Global filters on the python warnings. Using this since filter
 # behaviour is a bit weird.
 SERIALIZATIONWARNING = True
-KWARGSNUMERICWARNING1 = True
-KWARGSNUMERICWARNING2 = True
+KWARGSNUMERICWARNING1 = False
+KWARGSNUMERICWARNING2 = False
+
+# Exclude these parameters from the image metadata. These are rarely sampled
+# and would bloat the metadata or make it hard to serialize.
+EXCLUDE_FROM_METADATA = (
+    # Long list of sources that were included
+    ('source_parameters', 'source_inclusion_list'),
+    # Big array denoting the PSF
+    ('psf_parameters', 'kernel_point_source'),
```

Comment:
I think we want to keep the psf information, since generically we may have a variable PSF